src/public/css/style.css

```diff
@@ -4,7 +4,36 @@ body {
   font-family: prociono;
   font-size: 15px;
 }
+  /*<style>*
+a:link    {color:green; background-color:transparent; text-decoration:none}
+a:visited {color:pink; background-color:transparent; text-decoration:none}
+a:hover   {color:red; background-color:transparent; text-decoration:underline}
+a:active  {color:yellow; background-color:transparent; text-decoration:underline}*/
+/*</style>*/
+
+a:link {
+  color: white;
+  background-color: transparent;
+  text-decoration: none;
+}
+
+a:link:visited {
+  color: white;
+  background-color:transparent;
```

Comment:
there are no spaces between `:` and the value
`background-color:transparent;` should be
`background-color: transparent;`